packages/lib/src/types/task.ts

```diff
@@ -48,6 +48,27 @@ export enum TaskStatus {
   KnownIssue = "known-issue",
 }
 
+// Task status sorted in order of "relevance", with the most likely actionable at the top.
+export const SortedTaskStatus = [
+  TaskStatus.Failed,
+  TaskStatus.TestTimedOut,
+  TaskStatus.TaskTimedOut,
+  TaskStatus.KnownIssue,
+  TaskStatus.SetupFailed,
+  TaskStatus.SystemFailed,
+  TaskStatus.SystemTimedOut,
+  TaskStatus.SystemUnresponsive,
+  TaskStatus.Started,
+  TaskStatus.Dispatched,
+  TaskStatus.Succeeded,
+  TaskStatus.Undispatched,
+  TaskStatus.WillRun,
+  TaskStatus.Aborted,
+  TaskStatus.Blocked,
+  TaskStatus.Unscheduled,
+  TaskStatus.Inactive,
```

Comment:
Since you removed that display status logic you can probably remove `Inactive` now too!

<img width="577" alt="Screenshot 2024-11-25 at 4 13 49 PM" src="https://github.com/user-attachments/assets/9d0dbb69-433a-4aa9-82d6-ab686f82802c">